api/webendpoints_test.go

```diff
@@ -0,0 +1,141 @@
+package api
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"sync"
+	"testing"
+
+	dprequest "github.com/ONSdigital/dp-net/request"
+	"github.com/ONSdigital/dp-topic-api/config"
+	"github.com/ONSdigital/dp-topic-api/mocks"
+	"github.com/ONSdigital/dp-topic-api/store"
+	storetest "github.com/ONSdigital/dp-topic-api/store/datastoretest"
+
+	"github.com/gorilla/mux"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+var (
+	mu          sync.Mutex
+	testContext = context.Background()
+)
+
+func TestPublishedSubnetEndpointsAreDisabled(t *testing.T) {
+
+	type testEndpoint struct {
+		Method string
+		URL    string
+	}
+
+	publishSubnetEndpoints := map[testEndpoint]int{
+		// Dataset Endpoints
+		/*{Method: "POST", URL: "http://localhost:22000/datasets/1234"}:                            http.StatusMethodNotAllowed,
+		{Method: "PUT", URL: "http://localhost:22000/datasets/1234"}:                             http.StatusMethodNotAllowed,
+		{Method: "PUT", URL: "http://localhost:22000/datasets/1234/editions/1234/versions/2123"}: http.StatusMethodNotAllowed,*/
```

Comment:
Are these needed? If not the comments should be removed.

Reply:
They might be in future work, as mentioned elsewhere.
As the work evolves i will know better and do a clean up then.
Thanks.